src/parser/test/ParserTest.cpp

```diff
@@ -1098,4 +1098,39 @@ TEST(Parser, ConfigOperation) {
     }
 }
 
+TEST(Parser, GroupBy) {
+    // All fun succeed
+    {
+        GQLParser parser;
+        std::string query = "GO FROM 1 OVER work "
+                            "YIELD $$.company.name, $^.person.name "
+                            "| GROUP BY $$.company.name "
+                            "YIELD $$.company.name as name, "
+                            "COUNT($^.person.name ), "
+                            "COUNT_DISTINCT($^.person.name ), "
+                            "SUM($^.person.name ), "
+                            "AVG($^.person.name ), "
+                            "MAX($^.person.name ), "
+                            "MIN($^.person.name ), "
+                            "F_STD($^.person.name ), "
+                            "F_BIT_AND($^.person.name ), "
+                            "F_BIT_OR($^.person.name ), "
+                            "F_BIT_XOR($^.person.name )";
+
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    // All fun error, empty group name
+    {
+        GQLParser parser;
+        std::string query = "GO FROM 1 OVER work "
+                            "YIELD $$.company.name, $^.person.name "
+                            "| GROUP BY "
+                            "YIELD $$.company.name as name, "
+                            "COUNT($^.person.name )";
+        auto result = parser.parse(query);
+        ASSERT_FALSE(result.ok());
```

Comment:
if it's syntax error, better to assert syntax error than just false

Reply:
This is the parser module, and the result is not ok, it means syntax error